apps/example-orchestrator-ui/components/Subscriptions/Subscriptions.tsx

```diff
@@ -180,7 +182,11 @@ export const Subscriptions: FC<SubscriptionsProps> = ({
     );
     const queryContainsInvalidParts =
         filterQueryTupleArray?.includes(undefined);
-    const filterBy = filterQueryTupleArray?.filter(isValidQueryPart);
+    const filterBy =
+        filterQueryTupleArray
+            ?.concat([alwaysOnFilter])
+            .filter(isValidQueryPart) ||
+        (alwaysOnFilter ? [alwaysOnFilter] : undefined);
```

Comment:
is this check needed? empty array should basically be the same as undefined.

Reply:
ohh, I see, I do believe it can be improved by removing the special undefined logic. change alwaysOnFilter to have default `alwaysOnFilter=[]` and not be undefined, the backend is fine with an empty array.

Reply:
It would be better to change the `alwaysOnFilter` to accept multiple filters

Reply:
done, supporting multiple alwaysOnFilters now too